servers/src/test/java/tachyon/worker/block/BlockMetadataManagerTest.java

```diff
@@ -216,7 +266,7 @@ public void moveBlockMetaTest() throws Exception {
   }
 
   @Test
-  public void moveBlockMetaExceedCapacity() throws Exception {
+  public void moveBlockMetaExceedCapacityDeprecated() throws Exception {
```

Comment:
oh okay, makes sense. Could you suffix the method with `Test` (our test naming convention).